clients/fides-js/src/fides.ts

```diff
@@ -117,6 +123,7 @@ async function init(this: FidesGlobal, config: FidesConfig) {
   });
   if (initialFides) {
     Object.assign(this, initialFides);
+    updateWindowFides(this);
```

Comment:
Our synthetic GTM event (https://github.com/ethyca/fides/pull/4847/files#diff-a04bb0647e72a17425850d5589eee0a2f72b4f239a0d721899cd15029555be6aR62) requires window.Fides to contain up-to-date properties

Reply:
This is such a subtle footgun! Gah, the state mgmt in FidesJS has a lot of footguns

Reply:
agreed. This is the flakiest part of FidesJS